frontend/src/libs/components/router/router.tsx

```diff
@@ -1,34 +1,31 @@
+import { Route, Routes } from 'react-router-dom';
+
 import { AppRoute } from '~/libs/enums/enums.js';
 import { Auth } from '~/pages/auth/auth.js';
 import { NotFound } from '~/pages/not-found/not-found.js';
 
 import { App } from '../app/app.js';
+import { ProtectedRoute } from '../components.js';
 import { RouterProvider } from '../router-provider/router-provider.js';
 
 const Router = (): JSX.Element => (
   <RouterProvider
     routes={[
```

Comment:
There is no need to use routes={} since you have added <Routes>
```
<RouterProvider>
<Routes>
...
</Routes>
<RouterProvider/>
```

Reply:
I found only one way to get rid of the wrapper, via `createRoutesFromElements`, but for that I had to remove `<Routes></Routes>` so that this method could build the routes itself.